impeller/typographer/backends/skia/typographer_context_skia.cc

```diff
@@ -147,8 +147,8 @@ static ISize OptimumAtlasSizeForFontGlyphPairs(
           Allocation::NextPowerOfTwoSize(current_size.width + 1),
           Allocation::NextPowerOfTwoSize(current_size.height + 1));
     }
-  } while (current_size.width <= kMaxAtlasSize &&
-           current_size.height <= kMaxAtlasSize);
+  } while (current_size.width <= max_texture_size.width &&
+           current_size.height <= max_texture_size.height);
```

Comment:
Can you also add this change to the STB backend? It's similarly pinned to an arbitrary texture size.

Reply:
Filed https://github.com/flutter/engine/pull/46010